app/daemon/daemon.go

```diff
@@ -263,7 +265,17 @@ func (d *daemon) CreateGame(ctx context.Context, req *pb.CreateGameRequest) (*pb
 	return &pb.CreateGameResponse{Message: "Game is created"}, nil
 }
 func (d *daemon) StopGame(ctx context.Context, req *pb.StopGameRequest) (*pb.StopGameResponse, error) {
-	return nil, status.Errorf(codes.Unimplemented, "method StopGame not implemented")
+
+	game := req.Tag
+
+	g, ok := d.games[store.Tag(game)]
+	if !ok {
+		return &pb.StopGameResponse{}, fmt.Errorf("Error: game [ %s ] is not exists", g.Config().Tag)
+	}
+	if err := g.Close(); err != nil {
+		return &pb.StopGameResponse{}, nil
```

Comment:
Should it maybe return the error here?

Reply:
yeah true, it should return the error here, right :)

Reply:
```suggestion
		return &pb.StopGameResponse{}, fmt.Errorf("Error: error closing the game [ %s ]", g.Config().Tag)
```